libkbfs/tlf_journal.go

```diff
@@ -5,22 +5,39 @@
 package libkbfs
 
 import (
+	"errors"
 	"fmt"
 	"path/filepath"
 	"sync"
 
 	"github.com/keybase/client/go/logger"
-	"github.com/keybase/client/go/protocol/keybase1"
 	"golang.org/x/net/context"
 )
 
 // tlfJournalConfig is the subset of the Config interface needed by
 // tlfJournal (for ease of testing).
 type tlfJournalConfig interface {
+	BlockSplitter() BlockSplitter
 	Codec() Codec
 	Crypto() Crypto
-	KBPKI() KBPKI
+	currentInfoGetter() currentInfoGetter
+	encryptionKeyGetter() encryptionKeyGetter
```

Comment:
It might make more sense to embed these private interfaces into the `tlfJournalConfig` interface itself.